parlai/chat_service/core/manager_utils.py

```diff
@@ -345,13 +349,7 @@ def _world_function():
                     agent_state.onboard_data = onboard_data
                 self.manager.add_agent_to_pool(agent_state, world_type)
                 utils.print_and_log(logging.INFO, 'onboarding/overworld complete')
-                time.sleep(5)
 
-                # sleep until agent returns from task world
-                while agent_state.get_active_agent() != overworld_agent:
-                    time.sleep(2)
-                overworld.return_overworld()
```

Comment:
so now `return_overworld` is never called? should we remove these funcs from the demos?

Reply:
Yeah `return_overworld` was a function that we proposed for the initial messenger overworld API but never actually ended up needing. I can remove all callsites and defs